recipes/cyrsoxs/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "cyrsoxs" %}
+{% set version = "1.1.5.0" %}
+{% set sha = "7ce6e64d5e794751476e6e1b6dbb1cd940b2dd26796b8d416fd82b6c20302ff9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/usnistgov/cyrsoxs/archive/refs/tags/V{{ version }}.tar.gz
+  sha256: {{ sha }}
+
+build:
+  skip: True  # [py<37]
+  skip: True  # [not linux64]
+  skip: True  # [cuda_compiler_version == "None"]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake
+    - make
+    - openmp
+    - python >3.7
+    - pip
```

Comment:
```suggestion
```

why do you need python in build. can you try to remove python and pip from here

Reply:
I believe it needs Python so that pybind11 can link against it to produce the python module output... pip is definitely not needed, and I'm giving it a try in the local docker container without python also just to check.

Reply:
I believe pip in host is also just leftover from the template, now that I look at it.  Removing that as well.

Reply:
Works without python specified in build, updated.  Sorry!  I think pybind11 pulls it in as a dependency as it does wind up in the solved env.